scripts/eth_harvest.py

```diff
@@ -39,8 +39,6 @@
     ETH_CVX_CRV_HELPER_STRATEGY,
     ETH_BVECVX_STRATEGY,
     ETH_TRICRYPTO_STRATEGY,
-    ETH_TBTC_CRV_STRATEGY,
-    ETH_RENBTC_CRV_STRATEGY,
```

Comment:
If this is the Convex bcrvRenBTC then we should keep this. the RenBTC vault from the list was the already deprecated Harvest RenBTC Crv vault.